Resources/doc/versioning.rst

```diff
@@ -0,0 +1,177 @@
+API versioning
+==============
+Why version your API ?
+----------------------
+If you want to introduce breaking changes in your API, you should think about version your API. Indeed, if you introduce them on your current API, your users won't necessary upgrade their application directly and their applications won't work.
+
+How to version your API ?
+-------------------------
+There are several ways, none is standard :
+    - Use a query parameter ``/users?version=v1``
+    - Use a custom mime-type with an ``Accept`` header ``Accept: application/json; version=1.0``
+    - Use a custom header ``X-Accept-Version: v1``
+
+The ``FOSRestBundle`` allows you to use several of them at the same time or to choose one of them.
+
+URI API versioning
+------------------
+If you want to version your api with the uri, you can simply use the symfony router:
+
+.. code-block:: yaml
+
+    # app/config/routing.yml
+    my_route:
+        # ...
+        path: /{version}/foo/route
+
+Note: this will override the ``version`` attribute of the request if you use the ``FOSRestBundle`` versioning.
+
+Configure ``FOSRestBundle`` to use the api versioning
+-----------------------------------------------------
+You should activate the versioning in your config.yml:
+
+.. code-block:: yaml
+
+    # app/config/config.yml
+    fos_rest:
+        versioning: true
+
+If you do not want to allow all the methods described above, you should choose which version resolver to enable:
+
+.. code-block:: yaml
+configyml
```

Comment:
Oops, you're right, this was corrected in https://github.com/FriendsOfSymfony/FOSRestBundle/commit/1d1ce09d6f7737e1230cb32c75933dc720226eb9.